lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystemE2E.cs

```diff
@@ -169,6 +169,8 @@ public void TestCopyToLocalE2E()
             var helloFilePath = PathToFile(HelloFile);
             var blob = _container.GetBlockBlobReference(HelloFile);
             var tempFilePath = Path.GetTempFileName();
+            File.Delete(tempFilePath);
```

Comment:
Please document this change in the commit message or move it to another commit.

Reply:
Message updated.